README.md

```diff
@@ -1,6 +1,6 @@
-To run the report manually
+##To run the report manually
 
-$ python nova-bugs.py
+$ bash run-report.sh
```

Comment:
Actually, I meant 4 spaces before the $ (and one after it). MarkDown makes that a nice concole-font output that way.

But, it does not matter too much, the README needs to get updated more anyway.